src/commands/roll.rs

```diff
@@ -126,6 +129,13 @@ pub async fn roll(
         ),
     );
 
+    let roll_str = rolls_str(&rolls, mastery);
+    let mut message = format!("# {roll_str}");
+
+    // Display the roll name if provided
+    if let Some(ref roll_name) = name {
+        write!(message, " **Roll Name:** `{}`", roll_name).unwrap();
```

Comment:
The older versions of the bot used this kind of `x: y` output, but I got some feedback that that was harder to understand. So I've moved to a more contextual / conversational output, which is why I liked the idea of
```
@{username} rolled for `{name}`
```
I believe you can get the username like this:
```rust
let username = &ctx.author().name;
```

Reply:
Hello thanks for the feedback